fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java

```diff
@@ -2,47 +2,49 @@
 
 import de.adorsys.opba.fintech.api.model.generated.InlineResponse2003;
 import de.adorsys.opba.fintech.impl.config.TppAisClient;
-import de.adorsys.opba.fintech.impl.database.entities.UserEntity;
-import de.adorsys.opba.fintech.impl.service.mapper.Mapper;
+import de.adorsys.opba.fintech.impl.database.entities.SessionEntity;
+import de.adorsys.opba.fintech.impl.mapper.ManualMapper;
 import de.adorsys.opba.fintech.impl.service.mocks.TppListAccountsMock;
 import de.adorsys.opba.tpp.ais.api.model.generated.AccountList;
+import feign.FeignException;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
 public class AccountService {
+    @Value("${mock.tppais.listaccounts}")
+    String mockTppAisString;
 
     private final TppAisClient tppAisClient;
 
-    public InlineResponse2003 listAccounts(ContextInformation contextInformation, UserEntity userEntity, String bankId) {
+    public InlineResponse2003 listAccounts(ContextInformation contextInformation, SessionEntity sessionEntity, String bankId) {
```

Comment:
same recommendation: provide named object in OpenAPI

Reply:
yes I can do with 240